lib/components/Utilities/Icon/index.tsx

```diff
@@ -10,16 +10,13 @@ import {
 const iconVariants = cva("inline-block shrink-0", {
   variants: {
     size: {
-      xl: "h-12 w-12",
-      lg: "h-8 w-8",
-      md: "h-5 w-5",
-      sm: "h-4 w-4",
-      xs: "h-3 w-3",
+      lg: "w-[24px]",
```

Comment:
What if you use [size](https://tailwindcss.com/docs/size)? That will set width and height, so no need for the `aspect-square`?

Reply:
Ooh didn't know that one, great!

Reply:
Let's better use `w-6` with `aspect-square`, because `size` doesn't seem to be using `rem`